modules/creating-image-expiration-notification.adoc

```diff
@@ -0,0 +1,74 @@
+// module included in the following assemblies:
+
+// * use_quay/master.adoc
+// * quay_io/master.adoc
+
+:_content-type: CONCEPT
+[id="creating-image-expiration-notification"]
+= Creating an image expiration notification
+
+Image expiration event triggers can be configured to notify users through email, Slack webhooks, and so on, and can be configured at the repository level. Triggers can be set for images expiring in any amount of days, and can work in conjunction with the auto-pruning feature.
+
+Image expiration notifications can only be set by using the {productname} v2 UI.
```

Comment:
We do have an existing `createRepoNotification` endpoint that can be used to create too.